rdflib/plugins/shared/jsonld/util.py

```diff
@@ -13,7 +13,8 @@
         import simplejson as json
 
 from io import TextIOBase, TextIOWrapper
-from posixpath import normpath, sep
+from os import sep
+from posixpath import normpath
```

Comment:
The problem with this is, os.sep is platform dependent, while posixpath is not.

On Linux:

```console
$ uname -a
Linux teekai.zoic.eu.org 6.0.8-200.fc36.x86_64 #1 SMP PREEMPT_DYNAMIC Fri Nov 11 15:03:58 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
$ python -m pydoc os | grep -E '^\s+sep = '
    sep = '/'
$ python -m pydoc posixpath | grep -E '^\s+sep = '
    sep = '/'
```

On Windows:

```console
$ uname -a
MINGW64_NT-10.0-19044 DESKTOP-KLQG489 3.1.7-340.x86_64 2021-10-12 16:29 UTC x86_64 Msys
$ python -m pydoc os | grep -E '^\s+sep = '
    sep = r'\'
$ python -m pydoc posixpath | grep -E '^\s+sep = '
    sep = '/'
```

If this is causing problems, which it seems it is, we should rather just use a private constant for it, like `_posixpath_sep = '/'`

However I still don't understand how you get this issue. python 3.7 to 3.11 has sep as part of posixpath as far as I can tell. It would be helpful if you could give an example of how your system was setup and where python came from and what version it is.

Reply:
I would change it but you did not give maintainers permission to modify your branch, if you do I will update your branch.